src/composables/useEthers.ts

```diff
@@ -74,10 +76,41 @@ async function activate(externalProvider: ExternalProvider) {
   network.value = _network
   address.value = _address
   balance.value = _balance.toBigInt()
+  // Put it outside the timer as the lookup method can occasionally take longer than 5000ms
+  dnsAlias.value = await lookupDNS(_network?.chainId, _address)
 
   isActivated.value = true
 }
 
+/**
+ * @title Lookup DNS Alias such as ENS
+ * @returns string
+ */
+async function lookupDNS(
+  chainId: number,
+  address: string,
+  _provider?: Web3Provider,
+) {
```

Comment:
How about if another parameter is added here (`service`)? To specify a web3 domain service? ENS can be specified by default, but others can be added later on such as Unstoppable Domains, Punk Domains etc.

Reply:
Good advice. This function serves to output the return value from the common/default DNS set. Providing more custom slots would be excellent, maybe I'll tweak it later.